examples/ethane.toml

```diff
@@ -0,0 +1,41 @@
+# run via:
+# cargo run ethane.toml --release
```

Comment:
I believe both examples run the same simulation? If so the `.rs` file could just read this input and run the corresponding simulation.